test-utils/testlib/src/fees_utils.rs

```diff
@@ -196,7 +196,15 @@ impl FeeHelper {
                         + self.cfg.fee(ActionCosts::function_call_byte).send_fee(sir)
                             * num_bytes as u64
                 }
-                Action::Transfer(_) => self.cfg.fee(ActionCosts::transfer).send_fee(sir),
+                Action::Transfer(_) => {
+                    if is_receiver_implicit {
+                        self.cfg.fee(ActionCosts::transfer).send_fee(sir)
+                            + self.cfg.fee(ActionCosts::create_account).send_fee(sir)
+                            + self.cfg.fee(ActionCosts::add_full_access_key).send_fee(sir)
```

Comment:
Huh, do I understand correctly this is effectively making any transfer to an implicit account via a meta transaction to include a fee for creating a (named?) account? What is the motivation behind this? Isn’t that not necessary today? What happens if the implicit account already exists and the intent is to test a basic transfer from one account via a meta transaction?

Reply:
Yeah that's unfortunately already what happens today on mainnet, regardless of meta transactions.
I just learned about this this week, which lead me to write this comment on another NEP: https://github.com/near/NEPs/pull/448#issuecomment-1429551068
Others seemed also somewhat surprised. ^^

This test util (which I added very recently) was just wrong / incompatible with implicit accounts.

Motivation: You don't know ahead of time if you need to create it or not but you must inlcude enough gas to cover all cases.
I hope we can get around to fix the implicit account creation flow and make it go through `CreateAccount` instead of (ab)using the transfer action for it, which would allow dropping the extra cost here. See also https://github.com/near/NEPs/issues/462